routes/web.php

```diff
@@ -1,16 +1,17 @@
 <?php
 
-/*
-|--------------------------------------------------------------------------
-| Web Routes
-|--------------------------------------------------------------------------
-|
-| Here is where you can register web routes for your application. These
-| routes are loaded by the RouteServiceProvider within a group which
-| contains the "web" middleware group. Now create something great!
-|
-*/
-
 Route::get('/', function () {
-    return view('welcome');
+    return view('map');
```

Comment:
What happens if, later, you don't want the map to be your root page? I would consider calling this "home" or "dashboard" or something so that your redirects around the app are more clear. Not a must-do, just something to consider :)